lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -130,13 +211,14 @@ def render_to_fragment(self, request, program_uuid, **kwargs):  # lint-amnesty,
 
         context = {
             'urls': urls,
-            'user_preferences': get_user_preferences(request.user),
+            'user_preferences': get_user_preferences(user),
             'program_data': program_data,
             'course_data': course_data,
             'certificate_data': certificate_data,
             'industry_pathways': industry_pathways,
             'credit_pathways': credit_pathways,
-            'program_discussions_enabled': program_discussions_is_enabled()
+            'program_discussions_enabled': program_discussions_is_enabled(),
```

Comment:
store `program_discussions_is_enabled()` in a variable and use it at all the places needed.

Reply:
It is only used once in this method